src/eradius_counter.erl

```diff
@@ -90,7 +90,7 @@ aggregate({Servers, {ResetTS, Nass}}) ->
 %% the other histogram things in eradius, but prometheus.erl will do it for us
 observe(Name, {{ClientName, ClientIP, _}, {ServerName, ServerIP, ServerPort}} = MetricsInfo, Value, Help) ->
     case code:is_loaded(prometheus) of
-        true ->
+        {file, _} ->
```

Comment:
One of those (is_xxx not returning true/false).
Another is dets:is_dets_file/1. Returns true/{error, Reason}.